src/nvim/tui/tui.c

```diff
@@ -1810,6 +1843,12 @@ static void patch_terminfo_bugs(TUIData *data, const char *term, const char *col
   data->unibi_ext.get_bg = (int)unibi_add_ext_str(ut, "ext.get_bg",
                                                   "\x1b]11;?\x07");
 
+  // To query support for the kitty keyboard protocol, request the terminal's
+  // support for kitty's progressive enhancement (CSI ? u) followed by a request
+  // for the primary device attributes (CSI c)
+  data->unibi_ext.get_extkeys = (int)unibi_add_ext_str(ut, "ext.get_extkeys",
+                                                       "\x1b[?u\x1b[c");
```

Comment:
I think it is easier to understand using timer than using response of `DA`.

Reply:
This is how the spec says to query for support: https://sw.kovidgoyal.net/kitty/keyboard-protocol/#detection-of-support-for-this-protocol

Reply:
I understand.